randovania/games/super_metroid/json_data/header.json

```diff
@@ -323,19 +323,14 @@
                 "description": "By taking damage during a screen transition and using x-ray, you can trigger this.",
                 "extra": {}
             },
-            "GravJump": {
-                "long_name": "Grav Jump",
-                "description": "",
-                "extra": {}
-            },
             "HBJ": {
                 "long_name": "Horizontal Infinite Bomb Jump",
-                "description": "",
+                "description": "By timing bombs precisely, Samus can gain an arbitrary amount of horizontal distance without touching the ground.",
                 "extra": {}
             },
             "MidAirMorph": {
                 "long_name": "Mid-Air Morph",
-                "description": "",
+                "description": "Morphing while in the air allows Samus to get into elevated morph ball tunnels without needing spring ball or bombs.",
```

Comment:
*Morph Ball, capitalized please
same for Spring Ball and Bombs

Reply:
I'll get into the habit of treating all items like this.